### DISCUSSION TITLE
How to fix  this issue ? 'process completed with exit code 137'

### DISCUSSION THREAD

**Discussion by shenalpigi** [community/community#68991] [Discussions]
### Select Topic Area

Question

### Body

how to fix this error regarding to memory?

**Comment by supuna97** [comment]
This is the method, see this how to solve this problem.
https://stackoverflow.com/questions/43268156/process-finished-with-exit-code-137-in-pycharm

**Reply by shenalpigi** [reply]
thank you

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
